Source/Extensions/Blazorise.Video/Video.razor.cs

```diff
@@ -41,6 +51,7 @@ public override async Task SetParametersAsync( ParameterView parameters )
                     ProtectionHttpRequestHeaders = new { Changed = protectionHttpRequestHeadersChanged, Value = paramProtectionHttpRequestHeaders },
                     CurrentTime = new { Changed = currentTimeChanged, Value = paramCurrentTime },
                     Volume = new { Changed = volumeChanged, Value = paramVolume },
+                    YoutubeVideoSource = new { Changed = youtubeVideoSourceChanged, Value = paramYoutubeVideoSource }
```

Comment:
@stsrki @steal4life 
So this `YoutubeVideoSource` is an "option" of the underlying plyr library?

What if we instead added a `SourceType` enum instead?

`SourceType.Default`
`SourceType.Youtube`

and the user still defines the source in the same Source parameter, and we act accordingly to the defined SourceType internally?